modules/openxr/extensions/openxr_extension_wrapper.h

```diff
@@ -67,6 +61,8 @@ class OpenXRExtensionWrapper {
 	virtual void *set_swapchain_create_info_and_get_next_pointer(void *p_next_pointer) { return p_next_pointer; }
 	virtual void *set_instance_create_info_and_get_next_pointer(void *p_next_pointer) { return p_next_pointer; }
 
+	virtual void on_register_metadata() {}
```

Comment:
Let's start adding documentation / comments on the extension callbacks, it'll make it easier for contributors to figure out which to override and for what reasons.

Reply:
I think that is a solid idea, I'll find some time to add proper comments on each function.

Reply:
Ok, added a bunch of descriptions, let me know if you agree with it